version.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+// tinyGoVersion of this package.
+// Update this value before release of new version of software.
+const tinyGoVersion = "0.1.0"
```

Comment:
The current tag is `v0.1`, not `v0.1.0`. Go uses the former, semver uses the latter. I'm fine with both, but I think they should be aligned.

Reply:
I suggest we use `x.y.z` format to make it explicit.